src/pages/AddMembersPage/NewMemberList.test.tsx

```diff
@@ -0,0 +1,180 @@
+import { act, fireEvent, render, screen } from '@testing-library/react';
+import { client } from 'lib/gql/client';
+
+import { TestWrapper } from 'utils/testing';
+
+import { Group } from './AddMembersPage';
+import NewMemberList from './NewMemberList';
+
+const save = async () => {
+  return [];
+};
+
+const group: Group = {
+  id: 1,
+  name: 'test',
+};
+
+jest.mock('lib/gql/client', () => ({
+  client: { query: jest.fn() },
+}));
+
+beforeEach(() => {
+  jest.clearAllMocks();
+});
+
+describe('adding an existing org member will show an error if exists only', () => {
```

Comment:
describe blocks are usually used to describe the "context" or the situation within which a few test cases may be re-used.

So maybe you would use a describe for "when org member already exists"

test --->

"it displays an error"

think about reading the describe and the test description outloud together and they should make sense.

Reply:
the goal is to make this output super clear and readable

<img width="669" alt="Screenshot 2023-08-15 at 11 53 01 AM" src="https://github.com/coordinape/coordinape/assets/83605543/58e77c12-5460-4887-b8a0-35618aa645d0">